public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -415,7 +415,13 @@ export class QueryEditorRow<TQuery extends DataQuery> extends PureComponent<Prop
     const colour = type === 'warning' ? 'orange' : 'blue';
     const iconName = type === 'warning' ? 'exclamation-triangle' : 'file-landscape-alt';
 
-    const serializedWarnings = uniqueWarnings.map((warning) => warning.text).join('\n');
+    let serializedWarnings;
+    if (uniqueWarnings.length > 1) {
+      const listItems = uniqueWarnings.map((warning) => warning.text);
+      serializedWarnings = <List items={listItems} renderItem={(item) => <>{item}</>} />;
+    } else {
+      serializedWarnings = uniqueWarnings[0].text;
+    }
```

Comment:
May I ask why we need the `if` statement? Couldn't we just use the List either way?

Reply:
Good point, it's no longer necessary now that the list does not use obvious bullet points.